int32 publish_count
int32 front_distance
int32 back_distance
bool drive_front_ok
bool drive_back_ok

